orchestrator/careplancontributor/service.go

```diff
@@ -128,6 +128,18 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 		log.Info().Msgf("Redirecting to %s", s.frontendUrl)
 		http.Redirect(response, request, s.frontendUrl, http.StatusFound)
 	})
+
+	// Logout endpoint
+	mux.HandleFunc(basePath+"/logout", s.withSession(func(writer http.ResponseWriter, request *http.Request, session *user.SessionData) {
+		s.SessionManager.Destroy(writer, request)
+		// If there is a 'Referrer' value in the header, redirect to that URL
+		if referrer := request.Header.Get("Referrer"); referrer != "" {
```

Comment:
it's `Referer` (USA/UK English?): https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Reply:
Ahhh. That is US english, I have learnt how to spell with UK/SA english :P will fix. 'Color' also always gets me...